starport/templates/typed/stargate/x/{{moduleName}}/client/cli/query{{TypeName}}.go.plush

```diff
@@ -44,3 +44,35 @@ func CmdList<%= title(TypeName) %>() *cobra.Command {
 
     return cmd
 }
+
+func CmdGet<%= title(TypeName) %>() *cobra.Command {
```

Comment:
Maybe we could rename this command as `show` as well

Reply:
Yes, I like `show` more.

Reply:
solved by 4d69a84